python/packages/azurefunctions/agent_framework_azurefunctions/_entities.py

```diff
@@ -181,12 +221,29 @@ async def run_agent(
             logger.error(f"Error type: {type(exc).__name__}")
             logger.error(f"Full traceback:\n{error_traceback}")
 
+            # Create error message
+            error_message = DurableAgentStateMessage.from_chat_message(
+                ChatMessage(role="assistant", contents=[ErrorContent(message=str(exc), error_code=type(exc).__name__)])
+            )
+
+            # Create and store error response in conversation history
+            error_state_response = DurableAgentStateResponse(
+                json_type="response",
```

Comment:
What options can this support? I only see `response` being used. If only response, consider making it default? 
Alternatively, maybe make it `response` and `error` maybe and remove the `is_error` property?